cfgov/v1/preprocessed/css/molecules/text-introduction.less

```diff
@@ -0,0 +1,68 @@
+/* topdoc
+  name: Theme variables
+  family: cfgov-molecules
+  notes:
+    - "The following color and sizing variables are exposed,
+       allowing you to easily override them before compiling."
+  patterns:
+  - name: Colors
+    codenotes:
+      - |
+        @m-text-introduction_rule
+  tags:
+  - less
+*/
+@m-text-introduction_rule: @horizontal-rule;
+
+
+/* topdoc
+  name: Text Introduction
+  family: cfgov-molecules
+  patterns:
+    - name: Default example
+      markup: |
+        <div class="m-text-introduction">
+            <h1 class="m-text-introduction_heading">Heading</h1>
+            <h2 class="m-text-introduction_subheading">Subheading</h2>
+            <p class="m-text-introduction_body">Body content</p>
+            <a class="m-text-introduction_link">Link</a>
+        </div>
+      codenotes:
+        - |
+          Structural cheat sheet:
+          -----------------------
+          .m-text-introduction
+            h1.m-text-introduction_heading
+            h2.m-text-introduction_subheading
+            p.m-text-introduction_body
+            a.m-text-introduction_link
+  tags:
+    - cfgov-molecules
+*/
+
+.m-text-introduction {
+    &_heading,
+    &_intro,
+    &_body,
+    &_link-container {
+        margin-bottom: unit( @grid_gutter-width / @base-font-size-px, em );
+    }
+
+    &_intro {
+      .h2();
+    }
+
+    &_link {
+        .jump-link();
```

Comment:
Fixed in https://github.com/cfpb/cfgov-refresh/commit/5afd78007624224086b0eff5a9f8343fc5c2832e